### DISCUSSION TITLE
加密貨幣想法交流～～

### DISCUSSION THREAD

**Discussion by www0000596** [1101-GS4719-C/Python-Community-GS4719-C#216] [General]
嗨大家
好奇大家對加密貨幣的看法？？有已經進入幣圈投資的同學嗎或是有想要進入幣圈進行投資的？還是會觀望這個市場就好？？或是有想要收藏一些藝術家、創作家發行的NFT嗎？
https://technews.tw/2021/12/26/banks-and-cryptocurrencies/

**Comment by Jie2222** [comment]
可能以後吧，現在是窮學生

**Reply by www0000596** [reply]
真的，感同身受😅

**Comment by vvvioletta** [comment]
我覺得加密貨幣這件事可以從兩個方向來看：貨幣和投資。加密貨幣的初衷是透過去中心化的方式來達成普惠金融，但要稱之為「貨幣」的其中一個元素是價值穩定（才能達到價值儲存），現在的加密貨幣市場波動過大，因此要將它視為貨幣我認為不太適當（雖然有USDT等穩定幣，但他背後的儲備量也還是有受質疑之處）
但如果將加密貨幣看作投資部位，那就是一個風險很高、背後要研究的東西比較複雜的投資商品，又因為沒有法規，所以讓這個市場的水深上加深。
至於NFT我認為又是另一個話題了，因為他的性質和加密貨幣幾乎不一樣，其中有關連的可能就是NFT需要用ETH買這點吧。之後有找到適合的文章再分享～

**Reply by www0000596** [reply]
了解！感謝你提供的想法，目前也還在思考自己對這個商品的定義是什麼，的確每個人對的想法都不同，我也有聽過一個說法是他是一個很好的避險工具．最重要的還是自己要對這個商品有明確的想法與操作的邏輯～

**Comment by Jen-Siang** [comment]
我認為加密貨幣的生態圈還不完善，可能還會再多觀望

**Reply by www0000596** [reply]
我也是會再多加觀望，多了解一些😃

**Comment by Tomlord1122** [comment]
大家想要投資或買NFT之類的時候也要小心那個是不是詐騙網站，最好的辦法就是在發行人的twitter or instagram連結點進去最好，不然花了大錢買了個山寨品......

**Reply by www0000596** [reply]
感謝你的提醒！風險考量跟資料來源的正確性都是非常重要的的！！的確有許多網路犯罪手法出現，這也是監理科技必需要與時俱進的🥶

**Comment by EricAgumonJou** [comment]
我來分享一則前一陣子被炒過頭的加密貨幣：魷魚幣。
https://tw.news.yahoo.com/遭割韭菜-魷魚幣突暴跌-秒歸零-004551507.html

**Reply by EricAgumonJou** [reply]
![23a5cd2af6b8d05584d0653002efb577](https://user-images.githubusercontent.com/91121500/147630732-80f585d3-a844-47a8-9d67-eb794f2f18b6.jpeg)
這個例子超扯，一達歷史高點的五分鐘後，魷魚幣就突然從2856美元暴跌至0.0008美元，跌幅高達99.999999999%。

**Reply by www0000596** [reply]
真的...
被🈹️爆了...

**Comment by yangming881005** [comment]
我目前有投資大概10萬元，主要在Gamefi的項目，也有在玩幣安的現貨交易，小額合約(這個建議不要亂玩插一根針可能就會爆倉)，
pionex的網格機器人，也有買NFT，用的是BSC鍊、以太鍊、雪崩鍊，建議可以先從BSC鍊下手，因為目前BSC鍊上的GAS費比以太鍊便宜蠻多的，我覺得幣圈投資真的風險蠻高的有聽過很多被盜被騙的故事，所以如果真的要嘗試可以先從小額開始投資，然後千萬要小心不要亂點別人給的連結，很多詐騙的網站，小心不要變成韭菜也不要看到幣價暴漲就FOMO不然可能會被掛在山頂。

**Reply by www0000596** [reply]
謝謝你的經驗分享！
如何學會不FOMO真的是一堂相當重要的課！🙌🏻
這個資金的輪動感覺也蠻快的，有些幣可能都漲不起來，有些真的是翻倍在漲

**Reply by chia-kai-chang** [reply]
感謝分享，中肯經驗大推

**Comment by MAKERRRofKEVIN** [comment]
之前有在MOPCON聽過幣圈做技術的前輩分享，他們認為現在發展有一個問題是太多投機者的加入會增加加密貨幣的不確定性，從某些角度來講他們也會期待一下希望能夠出現泡沫破掉的情形清掉這些投機者，但又害怕會影響一般使用者的信任，最實際的方法還是像他們這樣親自出來分享，拉更多有志之士加入技術開發的行列

**Comment by cody910120** [comment]
投資一定有風險，基金投資有賺有賠，申購前應詳閱公開說明書
